src/components/subscriptions/tests/expiration/SubscriptionExpirationBanner.test.jsx

```diff
@@ -51,8 +51,9 @@ describe('<SubscriptionExpirationBanner />', () => {
     };
     render(<ExpirationBannerWithContext detailState={detailStateCopy} />);
     expect(screen.queryByRole('alert')).not.toBeNull();
+    const elem = await screen.findByText('Dismiss');
```

Comment:
good! I always try to use the findBy instead of getby for these if I suspect there is going to be a timing issue with finding the element